docs/system-settings/upgrading-the-software.md

```diff
@@ -28,14 +28,45 @@ and install it on the console like this:
 
 1. Copy the installer file to a USB drive and insert into the console.
 
-2. In the screen **tools menu**, stop the console software using the
+2. In the screen **Tools menu**, stop the console software using the
 **Shutdown Software** button at the bottom of the list.
 ![Tools Menu](/docs/images/Tools-Menu.png)
 
-3. Open the **tools menu** again, touch the **Folders** icon and navigate to
+3. Open the **Tools menu** again, touch the **Folders** icon and navigate to
 the drive where the upgrade file is found.
 
 4. Run the installer by double clicking/double tapping.
 
 5. When the installation has completed, shut down the console and
-restart it. This will allow any operating system updates to install.
\ No newline at end of file
+restart it. This will allow any operating system updates to install.
+
+## Upgrading panel firmware with USB Expert
+
+After upgrading the software it is sometimes necessary to upgrade the firmware in the various panels which make up the console control surfaces. You can do this using the Avolites USB Expert application which is installed on the console.
+
+> Sometimes the USB Expert application might be opened automatically after completion of a software upgrade, to prompt you to upgrade panel firmware.
+
+Use the following steps to upgrade panel firmware:
+
+1. In the screen **Tools menu**, click on **Control Panel** and select the **USB Expert Console** option.
+![Tools Menu](/docs/images/Tools-Menu-Control-Panel.png)
+
+2. You should see a list of the panels on the console in the left hand **Connected Panels** window (the list will vary depending on the console type).
+![USB Expert Panel Update](/docs/images/USB-Expert-Panel-Update.png)
+
+3. On the right hand side of the window select the **Service** tab and click on the **Enter Boot** button.
+
+4. After a short delay (maximum 2 minutes) all panels should be listed as being in "Boot Mode" showing a spanner icon in the left hand **Connected Panels** window.
```

Comment:
You do not need to enter boot, the panels will appear in the auto update list without being in boot. When you select update it will cycle through all the panels putting each one in boot and then performing the update. After the update has finished they will exit boot automatically.